### DISCUSSION TITLE
Your ko-fi account doesn't accept donations

### DISCUSSION THREAD

**Discussion by kaspagious** [mclemente/fvtt-module-polyglot#147] [General]
As a thank your for your quick response i wanted to thank you with a small donation. 
But for some unknown reason i got the error message : "This creator cannot receive PayPal payments at this time. "

https://drive.google.com/file/d/1gyLhmBVaKS0pv4WFZL0nptYglTFzT6Tt/view?usp=sharing

**Comment by mclemente** [comment]
Oops, my Paypal account was really old to the point I haven't input my ID when it was created and it got blocked because of that.
I've created a new one and it should accept donations now. Thank you for your kindness!

**Comment by kaspagious** [comment]
Indeed the problem is fixed. Have a nice hot coffee
